content/en/docs/tasks/debug-application-cluster/debug-running-pod.md

```diff
@@ -172,20 +170,172 @@ Use `kubectl delete` to remove the Pod when you're finished:
 kubectl delete pod ephemeral-demo
 ```
 
-<!--
-Planned future sections include:
+## Debugging using a copy of the Pod
+
+Sometimes Pod configuration options make it difficult to troubleshoot in certain
+situations. For example, `kubectl exec` won't work with a container image that
+doesn't include a shell or if an application crashes on startup. In these
+situations you can use `kubectl debug` to create a copy of the Pod with
+configuration values changed to aid debugging.
+
+### Copying a Pod while adding a new container
+
+Adding a new container can be useful when your application is running but not
+behaving as you expect and you'd like to add additional troubleshooting
+utilities to the Pod.
+
+For example, maybe your application's container images are built on busybox
+but you need debugging utilities not included in busybox. You can simulate
+this scenario using `kubectl run`:
+
+```shell
+kubectl run myapp --image=busybox --restart=Never -- sleep 1d
+```
+
+Run this command to create a copy of `myapp` named `myapp-copy` that adds a
+new Ubuntu container for debugging:
+
+```shell
+kubectl debug myapp -it --image=ubuntu --share-processes --copy-to=myapp-debug
+```
+
+```
+Defaulting debug container name to debugger-w7xmf.
+If you don't see a command prompt, try pressing enter.
+root@myapp-debug:/#
+```
+
+{{< note >}}
+
```

Comment:
nit: You can remove the line before and after the note text.

Reply:
nit:  You may want to add a single sentence to introduce the list of flag descriptions.

It may be helpful to provide more context about the `share-processes` flag.
What do you think about:


The `--share-processes` flag causes the Pod to share a single process namespace between all containers in a Pod.
A single process namespace lets a container view and signal processes from other containers in the same Pod.
You can identify processes from other containers using `ps`.
For more information about `shareProcessNamespace`, see the [PodSpec](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podspec-v1-core) API.

From the API reference:
```
Share a single process namespace between all of the containers in a pod. When this is set containers will be able to view and signal processes from other containers in the same pod,
```

Reply:
* Removed the extra space
* The single sentence to introduce the list of flag descriptions was what another reviewer suggested I replace with the `note` tags. 😃  I'll leave this for now.
* For `--share-processes` I've replaced the description of the flag with a short reason why you'd want to use it here, and then linked to the `ShareProcessNamespace` docs for full context. I like the way it turned out, PTAL.